### PR TITLE
Clusters overview info

### LINKED ISSUE #1233: Users should be informed that providing they kubeconfig is also possible by clicking the white box
**Description**
In **Clusters Overview**, when clicking on the **Connect cluster** button, a user can provide the kubeconfig using drag and drop, pasting in manually in the designated area and by clicking the white box.
The last option is not explicitly mentioned. A user should be informed about such possibility.

**Area**
* Kyma Dashboard

**Reasons**
A user should know all the options that allow adding the kubeconfig file. 

**Assignees**

@grego952 

**Attachments**
Zrzut ekranu 2022-04-8 o 08 09 22

### PR DESCRIPTION
**Pull request by grego952**
**Description**

Changes proposed in this pull request:

- Add info in the **Connect cluster** view that a user can also browse the computer in search for the kubeconfig file.

**Related issue(s)**
Resolves #1233

### CODE REVIEW

**Review by mrCherry97 — DISMISSED** [review]

File: core/src/i18n/en.yaml [hunk 1]
```diff
@@ -580,7 +580,7 @@ components:
   error-panel:
     error: An error occured. The component cannot be rendered.
   file-input:
-    drag-file: Drag file here
+    drag-file: Drag your file here or click to browse your computer
```
Comment (akucharska) [inline core/src/i18n/en.yaml:583]: Maybe something like: `Choose a file or drag it here` or `Drag your file here or click to upload`

**Review by akucharska — APPROVED** [review]

**Review by akucharska — CHANGES_REQUESTED** [review]
See above

**Review by majakurcius — APPROVED** [review]

**Review by akucharska — APPROVED** [review]

**Pull request merged**